apps/src/code-studio/components/progress/BulkLessonVisibilityToggle.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {getStore} from '../../../redux';
+import PropTypes from 'prop-types';
+import Button from '@cdo/apps/templates/Button';
+import FontAwesome from '../../../templates/FontAwesome';
+import ReactTooltip from 'react-tooltip';
+import {toggleHiddenLesson} from '@cdo/apps/code-studio/hiddenLessonRedux';
+import {unitCalendarLesson} from '../../../templates/progress/unitCalendarLessonShapes';
+import _ from 'lodash';
+import style from './bulk-lesson-visibility-toggle.module.scss';
+
+function BulkLessonVisibilityToggle({lessons, sectionId, scriptName}) {
+  const tooltipId = _.uniqueId();
+
+  return (
+    <div className={style.container}>
+      <Button
+        text="Show All Lessons"
+        icon="eye"
+        color={Button.ButtonColor.gray}
+        onClick={() =>
+          toggleHiddenLessons(scriptName, sectionId, lessons, false)
+        }
+      />
+      <Button
+        text="Hide All Lessons"
+        icon="eye-slash"
+        color={Button.ButtonColor.gray}
+        onClick={() =>
+          toggleHiddenLessons(scriptName, sectionId, lessons, true)
+        }
+      />
+      <span data-tip data-for={tooltipId}>
+        <FontAwesome icon="info-circle" className={style.infoTipIcon} />
+      </span>
+      <ReactTooltip id={tooltipId} effect="solid">
+        <p>
+          Make all lessons in this Unit visible or hidden for your students.
+        </p>
+      </ReactTooltip>
```

Comment:
`HelpTip` has a [Portal](https://www.npmjs.com/package/react-portal) here, should I use one too? i'm not sure what the benefit is

Reply:
Out of curiosity, why didn't you use `HelpTip` here? (I'm not sure what benefit a Portal gets you either though)

Reply:
It just doesn't look very good, and doesn't have a way to pass in options to override the default look. I suppose I also could have modified it to let me pass in classNames/icons stuff, would that make more sense?

Reply:
I think this is fine in that case! Was just curious